### PR TITLE
docs: DS-11957: Adds RHODS learning-resources tiles

### PR DESCRIPTION
**Pull request by bredamc**
Need to remove the existing source files for the RHODS Learning Resources tiles, before adding them here.

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@bredamc the CI failed with a following error:
```
09:11:10 #13 0.917 Validating quickstarts
09:11:10 #13 0.930 Quickstart validation error. Error occured in:  docs/quickstarts/openshift-ai-rhods-launch/openshift-ai-rhods-launch.yml
09:11:10 #13 0.932 panic: displayName: cannot be blank.
```

File: docs/quickstarts/openshift-ai-rhods-launch/openshift-ai-rhods-launch.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    order: 1
+  labels:
+    service-category/data-services: "true"
+    product/rhods: "true"
+spec:
+  version: 0.1
+  type:
+    text: Documentation
+    color: orange
+    displayName: Learn how to launch Red Hat OpenShift Data Science
```
Comment (Hyperkid123) [inline docs/quickstarts/openshift-ai-rhods-launch/openshift-ai-rhods-launch.yml:16]: The `displayName` should be an attribute of the `spec` object. (There are two extra spaces that are not required)

Comment (bredamc) [reply docs/quickstarts/openshift-ai-rhods-launch/openshift-ai-rhods-launch.yml]: Many thanks for the info @Hyperkid123  -- fixed now. 
Should I have been able to see that CI error message myself?

**Comment by Hyperkid123** [comment]
@bredamc there is an issue with the seed job. Seems like the quickstart is not created properly. I'll run the changes locally and try to see what is wrong.

**Comment by bredamc** [comment]
Maybe it's the _annotations_ and _labels_ sections -- are they supported in this type of quickstart?

File: docs/quickstarts/openshift-ai-rhods-related-documentation/openshift-ai-rhods-related-documentation.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-related-documentation
```
Comment (Hyperkid123) [inline docs/quickstarts/openshift-ai-rhods-related-documentation/openshift-ai-rhods-related-documentation.yml:4]: This name does not match the name in the metadata. It should be `openshift-ai-rhods-related-documentation`

File: docs/quickstarts/openshift-ai-rhods-fasttrack/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+kind: QuickStarts
+name: openshift-ai-rhods-fasttrack
+tags:
+  - kind: bundle
+    value: openshift-ai
```
Comment (Hyperkid123) [inline docs/quickstarts/openshift-ai-rhods-fasttrack/metadata.yml:5]: Currently, there is no `openshift-ai` bundle in HCC. The bundles are named based on the URL fragment.

Where in HCC do you expect the tile should be visible?

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**